language/stdlib/modules/sliding_nonce.move

```diff
@@ -0,0 +1,45 @@
+address 0x0 {
+module SlidingNonce {
+    use 0x0::Transaction;
+
+    resource struct T {
```

Comment:
Would you mind adding some of the commentary from your excellent doc explaining this scheme as comments?

Reply:
This is great; thank you!